### PR TITLE
jo: add new package

### PR DESCRIPTION
**Pull request by 1480c1**
Signed-off-by: Christopher Degawa <ccom@randomderp.com>

### CODE REVIEW

**Comment by 1480c1** [comment]
bash-completion is in the wrong place

**Comment by 1480c1** [comment]
left bash-completion stuff up to the msys2 version of the package

File: mingw-w64-jo/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Maintainer: Christopher Degawa <ccom@randomderp.com>
+
+_realname=jo
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=1.4
+pkgrel=0
```
Comment (Biswa96) [inline mingw-w64-jo/PKGBUILD:7]: I think not everything start with zero, right? 🤔

Comment (1480c1) [reply mingw-w64-jo/PKGBUILD:7]: https://wiki.archlinux.org/index.php/PKGBUILD#pkgrel
> The release number. This is usually a positive integer number ... When a new version of the software is released, this value must be reset to 1

You're right

**Comment by lazka** [comment]
There was a plan at one point to make bash-completion also look in the mingw prefix, so this is fine actually.

**Pull request merged**